src/answers-umd.js

```diff
@@ -259,52 +259,28 @@ class Answers {
       if (parsedConfig.templateBundle) {
         this.renderer.init(parsedConfig.templateBundle);
       }
-
-      this._handlePonyfillCssVariables(
-        parsedConfig.disableCssVariablesPonyfill,
-        this._invokeOnReady.bind(this));
-      return this;
+    } else {
+      // Templates are currently downloaded separately from the CORE and UI bundle.
+      // Future enhancement is to ship the components with templates in a separate bundle.
+      this.templates = new DefaultTemplatesLoader(templates => {
+        this.renderer.init(templates);
+      });
     }
 
-    // Templates are currently downloaded separately from the CORE and UI bundle.
-    // Future enhancement is to ship the components with templates in a separate bundle.
-    this.templates = new DefaultTemplatesLoader(templates => {
-      this.renderer.init(templates);
-      this._handlePonyfillCssVariables(
-        parsedConfig.disableCssVariablesPonyfill,
-        this._invokeOnReady.bind(this));
+    this.promise = new Promise((resolve, reject) => {
```

Comment:
Does this stop us from being backwards compatible? It seems like people _must_ use the Promise with these changes. Is this what we wanted, considering a lot of current customers are using a non-promised version?

Reply:
Nope, it's still backwards compatible with onReady in the config. Right before the promise is resolved onReady is executed, even you if aren't using the promise version.

Reply:
Love the new promises around css-vars-ponyfill